packages/wp-story-editor/src/components/documentPane/status/status.js

```diff
@@ -101,6 +104,7 @@ function StatusPanel() {
       value: 'private',
       label: __('Private', 'web-stories'),
       helper: __('Visible to site admins & editors only', 'web-stories'),
+      disabled: isUploading && visibility !== 'private',
```

Comment:
Nit: Might be nice to move all these status strings to constants.

Reply:
Also wondering if all these should be disabled when uploading -- for consistency, even though it doesn't practically make a difference, thoughts?

Reply:
Hmm good question. I think I lean towards not disabling if not really needed.